src/Kernel/CompiledCode.class.st

```diff
@@ -571,6 +586,12 @@ CompiledCode >> primitiveErrorVariableName [
 	^nil
 ]
 
+{ #category : #testing }
+CompiledCode >> propertyAt: aString ifAbsent: aFullBlockClosure [ 
+	
+	^ aFullBlockClosure value
```

Comment:
Is thins for polymorphism with something else?

I find it strange to have the concept of properties and just evaluate the if absent block.

Reply:
Yes, the compiled methods have properties but the blocks do not.